src/Core/Utilities/UrlHelper.cs

```diff
@@ -0,0 +1,81 @@
+﻿using System;
+using System.Collections.Generic;
+using System.Linq;
+using System.Text;
+using Flurl;
```

Comment:
Flurl is an http client, right? Is it needed here?

Reply:
Nope, this one contains only the Url builder. Http client is served with `Flurl.Http`.

This is what I understand from their website : https://flurl.dev/
![image](https://user-images.githubusercontent.com/1884255/142896025-5b0345e7-b61b-41fe-9614-f3e589fcb715.png)

The native .net UrlBuilder seems not to provide any easy interface to manipulate QueryParams. I had the choice between constructing them manually or to use any known library like Flurl.

In fact for this specific `import` link it was not required, but I wanted to implement the same behavior than `CozyClient.GenerateWebLink` to be future proof.

Also I expect it to be useful when we will improve login url handling : https://trello.com/c/hJ6sa3Tn

Reply:
I don't want us to manipulate QueryParams / String by hands. Let's use a lib for that :)